tests/integration/offer/test_range.py

```diff
@@ -209,11 +209,11 @@ def test_range_is_reordable(self):
             product=product, category=included_category)
         self.range.included_categories.add(included_category)
 
-        self.range.invalidate_cached_ids()
+        self.range.invalidate_cached_queryset()
```

Comment:
I suppose `invalidate_cached_ids()` should remain as alias for `invalidate_cached_queryset()` to ensure backwards compatibility ?

Reply:
nah, this goes in 2.0 so we can cleanup and break stuff.

Reply:
Just FYI I'm in the middle of cutting a stable release for 2.0 (and we have already released a beta for it), so this change is going to have to go into 2.1. Will try and test/review in the next few days!